internal/app.go

```diff
@@ -231,7 +231,7 @@ func safeWrite(name string, data []byte) error {
 	return os.Rename(tmpfile, name)
 }
 
-func (a *App) extract(crypto CryptoHandler, config configSnapshot) error {
+func (a *App) extract(config configSnapshot) error {
```

Comment:
IIUC this is an unrelated cleanup, right?
May it go on its own too?

Reply:
1.22 complained so I bundled it with 1.22 changes

Reply:
This is what I suspected, and this is what distracts from the _real thing_.
That's why I'm asking to extract the 1.22 upgrade (plus this cleanup) to a separate PR.

Reply:
https://github.com/foundriesio/fioconfig/pull/47